### DISCUSSION TITLE
Change tab name in website

### DISCUSSION THREAD

**Discussion by popcake456** [sanidhyy/duolingo-clone#13] [Q&A] [answered]
first of all ... thank you for your guidance on drizzle .. it was overlapping with old drizzle and updated one

so last question .... how do i change the lingo tab name in website

**Comment by sanidhyy** [comment, marked as answer]
Hi @popcake456, Can you please describe your question briefly if you want to change tab name/title on all pages on any particular page? If you want to change lingo site title for all pages, you need to go to `config/index.ts` and change it from there.
